test/e2e/customconfigs/skipjobswithoutqueuename_test.go

```diff
@@ -412,5 +415,50 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 				}, util.LongTimeout, util.Interval).Should(gomega.Succeed())
 			})
 		})
+
+		ginkgo.It("should not suspend the pods created by a LeaderWorkerSet in the test namespace with queue-name label", func() {
```

Comment:
Please also add a test confirming the LWS is properly suspended when "queue-name" is not set.

Reply:
Done